### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by torn4dom4n**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (torn4dom4n) [inline README.md]: Enable macOS builds and Windows builds.

Comment (50Wliu) [inline README.md]: https://travis-ci.org/atom/language-puppet is already active.  I'll get back to you with Appveyor.

Comment (torn4dom4n) [inline README.md]: I added macOS Builds Status and Dependency Status

Comment (50Wliu) [inline README.md]: https://ci.appveyor.com/project/Atom/language-puppet

Comment (torn4dom4n) [inline README.md]: Can you give for me a link same this: https://ci.appveyor.com/api/projects/status/5t4pa451fu5e0ghg/branch/master?svg=true)

Comment (50Wliu) [inline README.md]: ![https://ci.appveyor.com/api/projects/status/github/atom/language-puppet?svg=true]
https://ci.appveyor.com/api/projects/status/github/atom/language-puppet?svg=true

Comment (torn4dom4n) [inline README.md]: Thanks!

**Pull request merged**